src/libraries/System.Net.ServicePoint/ref/System.Net.ServicePoint.cs

```diff
@@ -48,6 +48,7 @@ public partial class ServicePointManager
         public static int DefaultConnectionLimit { get { throw null; } set { } }
         public static int DnsRefreshTimeout { get { throw null; } set { } }
         public static bool EnableDnsRoundRobin { get { throw null; } set { } }
+        [System.Runtime.Versioning.UnsupportedOSPlatform("browser")]
```

Comment:
What about this property is unsupported?  It's just a simple getter.

Reply:
It is returning `EncryptionPolicy.RequireEncryption` which assembly `System.Net.Security` is unsupported on browser so the enum is unsupported too